_docs/quickstarts/scanner/index.md

```diff
@@ -14,14 +14,10 @@ Forseti Scanner is different from the Cloud Security Scanner, which does App
 Engine vulnerability scanning. Learn more about
 [Cloud Security Scanner](https://cloud.google.com/security-scanner/).
 
-## Scanner Rule Engines
+## Scanners
 
-Forseti Scanner currently runs a single scanner engine at a time. To see a list of 
-available engines, run:
-
-```bash
-$ forseti_scanner --list_engines
-```
+Forseti Scanner can run multiple scanners at a time. To configure which scanners
+to run, see [Configuring Forseti page: Configuring Scanner]({% link _docs/howto/configuring-forseti.md %}).
```

Comment:
I think the link should be a link to the section as well, e.g.

_docs/howto/configuring-forseti.md#configuring-scanner

you'll want to check that though